src/client/sandboxes/code-instrumentation/code-instrumentation.js

```diff
@@ -0,0 +1,41 @@
+import PropertiesInstrumentation from './properties/properties';
+import LocationInstrumentation from './location';
```

Comment:
If it's an object and not class, it should be lower cased: `locationInstrumentation`